packages/config/config/devices/0x0063/39348_54890_54891_zw4008.json

```diff
@@ -0,0 +1,97 @@
+// GE/Jasco 39348 /54890 /54891/ ZW4008
+// In-Wall Smart Switch
+{
+	"manufacturer": "GE/Jasco",
+	"manufacturerId": "0x0063",
+	"label": "39348 /54890 /54891/ ZW4008",
```

Comment:
Please normalize the slashes in labels. Preferably with a space before and after (to distinguish different device labels, config params are fine).

Reply:
Done